starport/services/network/path.go

```diff
@@ -0,0 +1,17 @@
+package network
+
+import (
+	"strconv"
+
+	"github.com/tendermint/starport/starport/pkg/xfilepath"
+)
+
+var (
+	// SpnPath returns the path used to store chain home from SPN
+	SpnPath = xfilepath.JoinFromHome(xfilepath.Path("spn"))
```

Comment:
Can we deprecate the xfilepath pkg and not use it here? We were using it before and it always felt awkward to me, I think it would be simpler if we just stick to the standard os and filepath packages.

Reply:
It was used in the `chain` package to avoid extensive error checking and it is useful in this sense

Maybe less necessary for this package